client/post-editor/drafts-button/index.jsx

```diff
@@ -11,6 +11,8 @@ import get from 'lodash/get';
 import Button from 'components/button';
 import Count from 'components/count';
 import { getSelectedSite } from 'state/ui/selectors';
+import { getAllPostCount } from 'state/posts/counts/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
```

Comment:
This can be consolidated with the import above for `getSelectedSite`, i.e.:

```js
import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
```